src/services/Frontline.js

```diff
@@ -124,6 +135,111 @@ class Frontline {
       }
     }
   }
+
+  /**
+   * Load Infinity
+   */
+
+  httpRequestWrapper = (method, URL) => {
+    return new Promise((resolve, reject) => {
+      const xhr_obj = new XMLHttpRequest();
+      xhr_obj.responseType = "json";
+      xhr_obj.open(method, URL);
+      xhr_obj.onload = () => {
+        const data = xhr_obj.response;
+        resolve(data);
+      };
+      xhr_obj.onerror = () => {
+        reject("failed");
+      };
+      xhr_obj.send();
+    });
+  };
+
+  async getProducts(apiEndpoint, params) {
+    return await this.httpRequestWrapper(
+      "GET",
+      `${apiEndpoint}/products.json?${params}`
+    );
+  }
+
+  footerHeight = () => {
+    var sum = 0;
```

Comment:
`const`

Reply:
It can't be `const` variable as we're altering it later.

Reply:
Then it's `let` ;)

Reply:
The reason being that `let` and `const` are lexically scoped while `var` is not - which leads to all sorts of unexpected behaviours once people are used to lexical scoping

https://www.w3schools.com/js/js_scope.asp

Reply:
Perfect 😀

Reply:
I fixed it 🙂